src/components/UriViewer.js

```diff
@@ -178,12 +179,13 @@ const UriViewer = _.flow(
         const metadata = await loadObject(googleProject, bucket, name)
         setMetadata(metadata)
       } else {
+        // TODO: change below comment after switch to DRSHub is complete
```

Comment:
it might be a good idea for us to add the ticket number here and make a note in the ticket to update this. I tend to forget these things but also, having it in the ticket means that whoever picks up the ticket will know to update this.

Reply:
Ah great idea to make sure this doesn't get forgotten. I also totally forgot to add a ticket number the PR name, I will update that as well!